src/main/java/logic/commands/workout/workoutsession/WorkoutSessionAdd.java

```diff
@@ -14,19 +15,21 @@
 public class WorkoutSessionAdd extends Command {
 
     @Override
-    public void execute(String[] inputs, ExerciseList exerciseList,
-                        String filePath, Storage storage, boolean[] hasEndedWorkoutSessions) {
+    public CommandResult execute(String[] inputs, ExerciseList exerciseList,
+                                 String filePath, Storage storage, boolean[] hasEndedWorkoutSessions) {
+        String showToUser = "";
```

Comment:
Should the variable name be a noun (ie. "result" or "message") instead of a verb?